kuma/core/pipeline/sass.py

```diff
@@ -0,0 +1,13 @@
+from pipeline.compilers import SubProcessCompiler
+
+
+class DebugSassCompiler(SubProcessCompiler):
+    output_extension = 'css'
+
+    def match_file(self, filename):
+        return filename.endswith(('.scss', '.sass'))
+
+    def compile_file(self, infile, outfile, outdated=False, force=False):
+        # as gulp is compiling .scss files when DEBUG=True, all we need to do
+        # here is give Pipeline the location of the pre-compiled .css file
+        return outfile
```

Comment:
Ah, good to know. I was assuming `outfile` was used to instruct Pipeline on file name/path for template injection, but apparently not. 😸 

Excellent call to piggy-back on existing `SASSCompiler`.